### ISSUE TITLE
containers fail building

### ISSUE THREAD

**Issue by srcansiz** [fedbiomed/fedbiomed#383]
In GitLab by @mvesin on Oct 4, 2022, 18:17

With current `master` branch (v3.5 + patches) containers fail building.

Following https://fedbiomed.gitlabpages.inria.fr/latest/user-guide/deployment/deployment-vpn/ at step
```bash
${FEDBIOMED_DIR}/scripts/fedbiomed_vpn build vpnserver mqtt restful researcher
```

... fails as soon as `base` image with
```bash
Step 3/9 : RUN cargo install --bin boringtun --version ~0.3.0 boringtun
 ---> Running in d943e35debd3
[...]
error: failed to compile `boringtun v0.3.0`, intermediate artifacts can be found at `/tmp/cargo-installTZDVGv`

Caused by:
  failed to parse manifest at `/root/.cargo/registry/src/github.com-1ecc6299db9ec823/once_cell-1.15.0/Cargo.toml`

Caused by:
  failed to parse the `edition` key

Caused by:
  this version of Cargo is older than the `2021` edition, and only supports `2015` and `2018` editions.
```

Similar errors occur for next container builds.

This issue was reported through Fed-BioMed support list by Sliman Heithem
